src/cloudai/schema/test_template/nemo_launcher/slurm_command_gen_strategy.py

```diff
@@ -133,6 +128,8 @@ def _generate_cmd_args_str(self, args: Dict[str, str], nodes: List[str]) -> str:
         arg_str_parts = []
         for key, value in args.items():
             formatted_key = f"+{key}" if key.startswith("env_vars.") else key
```

Comment:
Would it make sense to handle cmd_args and env_args separately and join them only in the late stage? Lines 130 and 131 duplicate the check, makes me think we could avoid that.

Reply:
It makes sense. I will do.